json.go

```diff
@@ -0,0 +1,100 @@
+package clover
+
+import (
+	"os"
+	"encoding/json"
+	"bufio"
+	"io/ioutil"
+	"errors"
+)
+
+// Exporting errors
+var (
```

Comment:
The first two errors, ErrImportPathUnreachable and ErrExportPathExists are duplicated of the os.ErrExists and os.ErrNotExists.
It is better to use errors from the standard Go library, when possible, avoiding to introduce duplicates.